### PR TITLE
DP-186

### PR DESCRIPTION
**Pull request by jhsenjaliya**
adding skipTrash option in Hive command line that will skip trash while remove table data from HDFS.

### CODE REVIEW

File: metastore/ivy.xml [hunk 1]
```diff
@@ -32,8 +32,8 @@
     <dependency org="org.apache.hive" name="hive-serde" rev="${version}"
                 conf="compile->default" />
     <dependency org="commons-dbcp" name="commons-dbcp" rev="${commons-dbcp.version}">
-      <exclude module="commons-pool" />
-      <exclude org="org.apache.geronimo.specs" module="geronimo-jta_1.1_spec"/>
+        <exclude module="commons-pool" />
```
Comment (jhsenjaliya) [inline metastore/ivy.xml:35]: There is no change here

File: metastore/src/java/org/apache/hadoop/hive/metastore/RawStore.java [hunk 2]
```diff
@@ -18,10 +18,6 @@
```
Comment (jhsenjaliya) [inline metastore/src/java/org/apache/hadoop/hive/metastore/RawStore.java:18]: All Change in this file is actually reverting back HIVE-5181,
This patch is for branch-0.12-alti, for trunk I ll have to create new patch

File: metastore/src/java/org/apache/hadoop/hive/metastore/RetryingRawStore.java [hunk 3]
```diff
@@ -29,10 +29,12 @@
 import org.apache.commons.logging.Log;
```
Comment (jhsenjaliya) [inline metastore/src/java/org/apache/hadoop/hive/metastore/RetryingRawStore.java:29]: All Change in this file is actually reverting back HIVE-5181
This patch is for branch-0.12-alti, for trunk I ll have to create new patch

File: metastore/src/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java [hunk 4]
```diff
@@ -28,12 +28,7 @@
 import java.lang.reflect.Proxy;
 import java.net.URI;
 import java.net.URISyntaxException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Random;
+import java.util.*;
```
Comment (raviprak-altiscale) [inline metastore/src/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java]: Please explicitly list all the dependencies. Using wildcards is seen as an anti-pattern.

**Comment by jhsenjaliya** [comment]
Community preferred configuration way, so this becomes invalid to merge.

**Pull request closed without merging**